pkg/kubectl/cmd/templates/templater.go

```diff
@@ -223,7 +223,13 @@ func flagsUsages(f *flag.FlagSet) string {
 			format = "   %s   " + format
 		}
 
-		fmt.Fprintf(x, format, flag.Shorthand, flag.Name, flag.DefValue, flag.Usage)
+		usage := flag.Usage
+
+		if len(flag.Deprecated) != 0 {
+			usage += fmt.Sprintf(" (DEPRECATED: %s)", flag.Deprecated)
```

Comment:
is this the standard template for showing a deprecation note?
/cc @kubernetes/sig-cli-pr-reviews

Reply:
It's part of the standard template for help menus in pflag. Reference: https://github.com/spf13/pflag/blob/3ebe029320b2676d667ae88da602a5f854788a8a/flag.go#L720. Should we keep it that way?

Reply:
seems correct, thanks. up to the kubectl maintainers. :+1: